handlers.go

```diff
@@ -120,7 +128,7 @@ func checkLoginStatus(w http.ResponseWriter, r *http.Request) *internal.User {
 		http.Error(w, "Invalid login session.", http.StatusBadRequest)
 		return nil
 	} else if id < 1 {
-		log.Println("Invalid user id -", err)
+		log.Println("Invalid user id:", id)
```

Comment:
keep it as '-' instad of ':' for consistency?

Reply:
Ohh! So here is the pattern ive been subconciously using:
- If its an error, place it after a "-"
- Otherwise, use a regular ":"

Reason being, most packages return errors that use ":" a lot. For example:

sql error [69696]: sql failed to do a thing

so then if i have a message after it looks weird, like this:

Error while getting user: sql error [69696]: sql failed to do a thing

but with a "-" its better:

Error while getting user - sql error [69696]: sql failed to do a thing